src/ts/component/block/text.tsx

```diff
@@ -702,12 +702,17 @@ const BlockText = observer(class BlockText extends React.Component<Props> {
 				saveKeys.push({ key: `arrowright, arrowdown` });
 			};
 		};
-
-		const twineOpen = [ '[', '{', '\'', '\"', '(' ];
+		[];
+		const twineOpen = [ '[', '{', '\'', '\"', '`', '(', '【', '「', '（', '“', '‘' ];
```

Comment:
This needs refactoring, we can leave only one map and use it's keys as opening symbols.